nvflare/private/fed/client/communicator.py

```diff
@@ -301,7 +302,8 @@ def auxCommunicate(self, servers, project_name, token, fl_ctx: FLContext, client
             while retry > 0:
                 try:
                     start_time = time.time()
-                    self.logger.info(f"Send AuxMessage to {project_name} server")
+                    # self.logger.info(f"Send AuxMessage to {project_name} server")
```

Comment:
we should remove the commented out lines here as well.

Reply:
Removed.